src/parser/whamm.pest

```diff
@@ -3,7 +3,7 @@
 // ==============================
 
 // supports top-level global declarations/initial assignments and probe definitions
-script = { SOI ~ (statement | fn_def)* ~ probe_def ~ (statement | fn_def | probe_def)* ~ EOI }
+script = { SOI ~ (statement | fn_def)* ~ probe_def ~ (fn_def | probe_def)* ~ EOI }
```

Comment:
I'd add back the ability to put a `statement` after a `probe_def` and figure out what the bug was that kept this from working.

Reply:
this is fixed in PR #72. My bad